docs/sources/alerting/set-up/configure-alertmanager/index.md

```diff
@@ -0,0 +1,36 @@
+---
+aliases:
+  - /docs/grafana/latest/alerting/configure-alertmanager
+  - /docs/grafana/latest/alerting/configure-alertmanager
+description: Configure Alertmanager
+keywords:
+  - grafana
+  - alerting
+  - set up
+  - configure
+  - Alertmanager
+title: Configure Alertmanager
+weight: 100
+---
+
+# Configure Alertmanager
+
+Configure Alertmanager for Grafana Alerting to group and manage alert rules, adding a layer of orchestration on top of the alerting engines.
+
+## Add a new external Alertmanager
+
+1. In the Grafana menu, click the Alerting (bell) icon to open the Alerting page listing existing alerts.
+2. Click **Admin** and then scroll down to the External Alertmanager section.
+3. Click **Add Alertmanager** and a modal opens.
+4. Add the URL and the port for the external Alertmanager. You do not need to specify the path suffix, for example, `/api/v(1|2)/alerts`. Grafana automatically adds this.
```

Comment:
@yuri-tceretian I remember our last customer call where this was more complicated than that, anything to add?

Reply:
This is next up after provisioning--i didn't write any of this - i jsut copied it as is from existing alertmanager docs. Yuri -- we can sit down and chat :)